Makefile

```diff
@@ -95,10 +95,25 @@ vagrant-sync-optional:
 vagrant-deploy: vagrant-sync-config vagrant-sync-build
 	export KUBECTL="cluster/kubectl.sh --core" && ./cluster/deploy.sh
 
-.release-functest:
+functest-release: kubevirt.yaml
+	make vagrant-deploy DOCKER_TAG=$(TAG) DEPLOY_MANIFEST=$$PWD/kubevirt.yaml
 	make functest > .release-functest 2>&1
 
-release-announce: .release-functest
+kubevirt.yaml: TAG=$(shell git describe)
```

Comment:
Why is the kubevirt.yaml make target declared 3 times?

Reply:
The first two are declaring variables for that target's scope.
the third time the target is getting defined.